nilearn/decomposition/dict_learning.py

```diff
@@ -0,0 +1,202 @@
+"""
+DictLearning
+"""
+
+# Author: Arthur Mensch
+# License: BSD 3 clause
+from ..input_data.base_masker import filter_and_mask
+
+import numpy as np
+
+from sklearn.externals.joblib import Memory
+
+from .._utils import as_ndarray
+
+
+from .canica import CanICA
+from .._utils.cache_mixin import CacheMixin
+
+from sklearn.linear_model import Ridge
+from sklearn.decomposition import MiniBatchDictionaryLearning
+from sklearn.decomposition.pca import RandomizedPCA
+
+class DictLearning(CanICA, MiniBatchDictionaryLearning, CacheMixin):
+    """Perform Dictionary Learning analysis on a CanICA initialization : yields more stable maps
```

Comment:
I know I am always saying this, but "DictLearning: performs dictionary learning." does not appear to be a crisp, high-yield explanation to me. Could we not add at least one sentence, in particular with regard to difference to PCA and ICA, perhaps?